app/src/main/java/com/hamdan/forzenbook/theme/Theme.kt

```diff
@@ -119,156 +87,15 @@ object ForzenbookTheme {
     val colors: ReplacementColors
         @Composable
         get() = LocalReplacementColors.current
+    val dimens: Dimens
+        @Composable
+        get() = LocalDimens.current
 }
 
-object PaddingValues {
-    /**
-     * 4.dp
-     */
-    val smallPad_1 = 4.dp
-
-    /**
-     * 8.dp
-     */
-    val smallPad_2 = 8.dp
-
-    /**
-     * 12.dp
-     */
-    val smallPad_3 = 12.dp
-
-    /**
-     * 16.dp
-     */
-    val medPad_1 = 16.dp
-
-    /**
-     * 20.dp
-     */
-    val medPad_2 = 20.dp
-
-    /**
-     * 24.dp
-     */
-    val medPad_3 = 24.dp
-
-    /**
-     * 28.dp
-     */
-    val largePad_1 = 28.dp
-
-    /**
-     * 32.dp
-     */
-    val largePad_2 = 32.dp
-
-    /**
-     * 36.dp
-     */
-    val largePad_3 = 36.dp
-
-    /**
-     * 40.dp
-     */
-    val largePad_4 = 40.dp
-}
-
-object TextSizeValues {
-    /**
-     * 8.sp
-     */
-    val small_1 = 8.sp
-
-    /**
-     * 12.sp
-     */
-    val small_2 = 12.sp
-
-    /**
-     * 16.sp
-     */
-    val small_3 = 16.sp
-
-    /**
-     * 20.sp
-     */
-    val med_1 = 20.sp
-
-    /**
-     * 24.sp
-     */
-    val med_2 = 24.sp
-
-    /**
-     * 28.sp
-     */
-    val med_3 = 28.sp
-
-    /**
-     * 32.sp
-     */
-    val large_1 = 32.sp
-
-    /**
-     * 36.sp
-     */
-    val large_2 = 36.sp
-
-    /**
-     * 40.sp
-     */
-    val large_3 = 40.sp
-}
-
-object IconSizeValues {
-    /**
-     * 20.dp
-     */
-    val small_1 = 20.dp
-
-    /**
-     * 40.dp
-     */
-    val small_2 = 40.dp
-
-    /**
-     * 60.dp
-     */
-    val med_1 = 60.dp
-
-    /**
-     * 80.dp
-     */
-    val med_2 = 80.dp
-
-    /**
-     * 120.dp
-     */
-    val large_1 = 120.dp
-
-    /**
-     * 140.dp
-     */
-    val large_2 = 140.dp
-
-    /**
-     * 160.dp
-     */
-    val big_1 = 160.dp
-
-    /**
-     * 180.dp
-     */
-    val big_2 = 180.dp
-
-    /**
-     * 200.dp
-     */
-    val giga_1 = 200.dp
-
-    /**
-     * 220.dp
-     */
-    val giga_2 = 220.dp
-
-}
+//val adaptiveDimens: Dimens = if (LocalConfiguration.current.screenWidthDp <= SMALL_SCREEN_DP) {
```

Comment:
Don't leave commented stuff in here

Reply:
yup forgot to remove that after showing you over discord, removed.